service/src/main/resources/application.yaml

```
@@ -8,3 +8,10 @@ ktor:
 storage:
   mongodb:
     connectionString: "$MONGODB_URL:mongodb://localhost:27017"
+  dropbox:
+    auth:
+      refreshToken: "$DROPBOX_REFRESH_TOKEN"
+      appKey: "$DROPBOX_APP_KEY"
+      appSecret: "$DROPBOX_APP_SECRET"
+    paths:
+      xSounds: "$DROPBOX_X_SOUNDS_PATH:/lebkuchenFM/x-sounds/"
```

Actually the path should be just `/xsounds/` since we use the Dropbox Apps integration it will be automatically put in the right LebkuchenFM name space.

---

We don't need to get this from env, let's just leave it as 
```suggestion
      xSounds: "/xsounds/"
```

---

are you sure about this namespace? Does not work like that for me.
Thats why I've added possibility to set whatever path from env.

---

That's how we did it until now. If it does not work for you then leave the env part, but please change the default value.